app/js/lib/reducers/country_specifics.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var getCountrySpecifics = require( '../country_specifics_repository' ).getCountrySpecifics;
+
+function countrySpecificValidation( state, action ) {
+	var newState;
+	if ( typeof state !== 'object' ) {
+		newState = getCountrySpecifics( 'DE' );
+	} else {
+		newState = Object.assign( {}, state );
+	}
+
+	switch ( action.type ) {
+		case 'UPDATE_ELEMENT_ATTRIBUTES':
+			newState = getCountrySpecifics( action.payload.countryCode );
+			return newState;
+		default:
+			return newState;
```

Comment:
The first code before the switch block only gets used here. So why not have a guard statement that returns the stuff from the previous comment, followed by this code before the switch block? Looks like you don't need a single assignment in this function.